tests/integration_test.rs

```diff
@@ -206,11 +206,18 @@ async fn test_policy_evaluator(
 }
 
 #[rstest]
+// TODO: uncomment once the actual policy is published
```

Comment:
reminder to uncomment this

Reply:
Fixed, the code is now testing the actual policy